lib/mintacoin/assets/crypto/stellar.ex

```diff
@@ -93,7 +111,16 @@ defmodule Mintacoin.Assets.Stellar do
     [trustline_operation, payment_operation]
   end
 
-  @spec trustline_operation(asset :: asset(), distributor_pk :: key()) :: change_trust()
+  @spec build_trustline_operation(issuer_pk :: key(), asset_code :: asset_code()) :: operations()
+  defp build_trustline_operation(issuer_pk, asset_code) do
+    asset = [code: asset_code, issuer: issuer_pk]
+
+    trustline_operation = trustline_operation(asset, nil)
+
+    [trustline_operation]
```

Comment:
@jhonatan-kmt I think we can avoid the variable setting here, what do you think?

```suggestion
    [trustline_operation(asset, nil)]
```

Reply:
Is fine for me

Reply:
In this case, I think you can do the same in the `build_asset_operations/4` function, what do you think?

Reply:
I like it, I will make the corresponding change